### DISCUSSION TITLE
90多抽的基尼奇

### DISCUSSION THREAD

**Discussion by keyuanya** [share121/inter-knot#5140] [原神]
小保底吃满歪刻晴，之后大保底20抽提前出了基尼奇
![Screenshot_20240917_191347_com miHoYo Yuanshen](https://github.com/user-attachments/assets/d4036510-87f6-4d52-8c32-eb7d3b45fd02)
这已经是近期最欧的了，相比于绝区零小保底开服至今三连歪（唯一一次UP池提前还是30多抽歪），崩铁飞霄吃满大保底，已经算是非常好的了😿
![172517665993916782cab2da0771a60231d5d250d55d50840337a90ecc133306d839e4c8d1666 0](https://github.com/user-attachments/assets/35b80a2a-5504-4801-b988-3a6af73bf03a)

**Comment by Kjlbzbl** [comment]
摸摸头，从没欧过的我，已经习惯了
